### PR TITLE
feat(DBSS): add new resource dbss_instance

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

add new resource dbss_instance

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
add new resource dbss_instance
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dbss' TESTARGS='-run TestAccInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dbss -v -run TestAccInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccInstance_basic
=== PAUSE TestAccInstance_basic
=== CONT  TestAccInstance_basic
--- PASS: TestAccInstance_basic (737.80s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dbss      737.913s
```

### CODE REVIEW

File: docs/resources/dbss_instance.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+* `subnet_id` - (Required, String, ForceNew) The subnet ID of the NIC.
+
+  Changing this parameter will create a new resource.
+
+* `security_group_id` - (Required, String, ForceNew) Specifies the ID of the security group.
+
+  Changing this parameter will create a new resource.
+
+* `product_id` - (Required, String, ForceNew) The product ID.
```
Comment (houpeng80) [inline docs/resources/dbss_instance.md:67]: replaced with flavor_id?

Comment (Jason-Zhang9309) [inline docs/resources/dbss_instance.md]: There is another parameter named `flavor`

File: huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The connection address.`,
+			},
+			"connect_ipv6": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The IPv6 address.`,
+			},
+			"created": {
```
Comment (ShiChangkuo) [inline huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go]:
```suggestion
			"created_at": {
```

File: huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go [hunk 3]
```diff
@@ -0,0 +1,567 @@
...
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The creation time`,
+			},
+			"expired": {
```
Comment (ShiChangkuo) [inline huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go]:
```suggestion
			"expired_at": {
```

File: huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_instance_test.go [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  name   = "subnet-default"
+  vpc_id = data.huaweicloud_vpc.test.id
+}
+
+data "huaweicloud_networking_secgroup" "test" {
+  name = "default"
+}
+
+data "huaweicloud_availability_zones" "test" {
+  region = "cn-north-4"
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_instance_test.go]: it's better to use **acceptance.HW_REGION_NAME**

File: docs/resources/dbss_instance.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+  Only letters, digits, underscores (_), and hyphens (-) are allowed.
+
+  Changing this parameter will create a new resource.
+
+* `availability_zone` - (Required, String, ForceNew) The availability zone to which the instnce belongs.
+  Primary and secondary AZs are separated by commas. Example: cn-north-4a,cn-north-4b.
+
+  Changing this parameter will create a new resource.
+
+* `flavor` - (Required, String, ForceNew) Specifies the flavor. Possible values are:
```
Comment (ShiChangkuo) [inline docs/resources/dbss_instance.md:50]: suggest to put `resource_spec_code, `flavor` and `product_id` together.

File: huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go]: fixed

File: huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go (hunk 3 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go]: fixed

File: huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_instance_test.go (hunk 4 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_instance_test.go]: fixed

File: docs/resources/dbss_instance.md (hunk 5 above)
Comment (Jason-Zhang9309) [reply docs/resources/dbss_instance.md:50]: fixed

**Pull request merged**